lib/timex.ex

```diff
@@ -633,7 +633,6 @@ defmodule Timex do
       {year, month, day} = Helpers.iso_day_to_date_tuple(year, iso_day)
       %Date{year: year, month: month, day: day}
   end
-  def from_iso_triplet(_, _, _), do: {:error, {:from_iso_triplet, :invalid_triplet}}
```

Comment:
Yeah, definitely. Missed guard clause in the upper clause, thought it would catch all triplets. Done.